app/auth.py

```diff
@@ -0,0 +1,57 @@
+from app import app
+from datetime import datetime, timedelta
+from flask import request, Response, session
+from functools import wraps
+import jwt
```

Comment:
good try! But please go through pep8 imports organisation. built-in imports should be at the top.